platform/ui/src/components/InputDoubleRange/InputDoubleRange.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useCallback, useEffect, useRef } from 'react';
+import classNames from 'classnames';
+import Typography from '../Typography';
+import './InputDoubleRange.css';
+
+/**
+ * React Range Input component
+ * it has two props, value and onChange
+ * value is a number value
+ * onChange is a function that will be called when the range input is changed
+ *
+ *
+ */
+
+const InputDoubleRange: React.FC<{
+  valueLeft: number;
+  valueRight: number;
+  onChange: (leftVal: number, rightVal: number) => void;
+  onSliderChange;
+  minValue: number;
+  maxValue: number;
+  step: number;
+  unit?: string;
+  containerClassName?: string;
+  inputClassName?: string;
+  labelClassName?: string;
+  labelVariant?: string;
+  showLabel: boolean;
+}> = ({
+  valueLeft,
+  valueRight,
+  onChange,
+  onSliderChange,
+  minValue,
+  maxValue,
+  step = 1,
+  unit = '',
+  containerClassName,
+  inputClassName,
+  labelClassName,
+  labelVariant,
+  showLabel = true,
+}) => {
+  const [leftVal, setLeftVal] = useState(valueLeft);
+  const [rightVal, setRightVal] = useState(valueRight);
+
+  const leftValRef = useRef(valueLeft);
```

Comment:
why do you need two refs to the initial valueLeft and valueRight, add documentation pelase